Gemfile.lock

```diff
@@ -819,4 +840,4 @@ DEPENDENCIES
   web-console (>= 3.3.0)
 
 BUNDLED WITH
-   1.15.2
+   1.15.3
```

Comment:
I've ended up downgrading bundler a couple of times to avoid this line in other PRs, but am happy to keep it.